src/core/MREntity.js

```diff
@@ -227,7 +238,7 @@ export class MREntity extends MRElement {
 
     rotation = {
         get: () => {
-            return this.dataset.rotation;
+            return mrjsUtils.string.stringToVector(this.dataset.position);
```

Comment:
typo, looks like a misplaced copy-paste - this should still have `this.dataset.rotation` inside the utils function

Reply:
lol good catch. that could've been bad.